types/theme-ui/index.d.ts

```diff
@@ -75,7 +75,7 @@ export type ColorMode = {
     accent?: CSS.ColorProperty;
 };
 
-export interface Theme extends Omit<StyledSystemTheme, 'colors'> {
+export interface Theme extends Omit<Omit<StyledSystemTheme, 'colors'>, 'buttons'> {
```

Comment:
What do you think about this? 👇

```suggestion
export interface Theme extends Omit<StyledSystemTheme, 'colors' | 'buttons'> {
```

Reply:
Yeah that's better. My TS inexperience is showing.